ctapipe/tools/train_energy_regressor.py

```diff
@@ -0,0 +1,156 @@
+import numpy as np
+
+from ctapipe.core import Tool
+from ctapipe.core.traits import Bool, Int, Path, flag
+from ctapipe.io import TableLoader
+from ctapipe.reco import CrossValidator, EnergyRegressor
+from ctapipe.reco.preprocessing import check_valid_rows
+
+__all__ = [
+    "TrainEnergyRegressor",
+]
+
+
+class TrainEnergyRegressor(Tool):
+    """
+    Tool to train a `~ctapipe.reco.EnergyRegressor` on dl1b/dl2 data.
+
+    The tool first performs a cross validation to give an initial estimate
+    on the quality of the estimation and then finally trains one model
+    per telescope type on the full dataset.
+    """
+
+    name = "ctapipe-train-energy-regressor"
+    description = __doc__
+
+    examples = """
+    ctapipe-train-energy-regressor \\
+        --config ml_config.yaml \\
+        --input gamma.dl2.h5 \\
+        --ouput energy_regressor.pkl
+    """
+
+    output_path = Path(
+        default_value=None,
+        allow_none=False,
+        directory_ok=False,
+        help="Ouput path for the trained reconstructor.",
```

Comment:
and here.  Also should mention the file type that is written, so the user knows what file extension to use.